proto/cacheclient.proto

```diff
@@ -466,62 +466,82 @@ message _SortedSetPutRequest {
 
 message _SortedSetPutResponse {}
 
+message _Unbounded {}
+
```

Comment:
should use the `_Unbounded` message in the ListFetch fetch PR https://github.com/momentohq/client-protos/pull/126